app/experimenter/experiments/forms.py

```diff
@@ -725,6 +725,21 @@ class Meta:
 
     def clean(self):
         cleaned_data = super().clean()
+        self._validate_risk(cleaned_data)
+        self._validate_risk_technical(cleaned_data)
+        return cleaned_data
+
+    def _validate_risk(self, cleaned_data):
+        if True in cleaned_data.values():
```

Comment:
Hmm! This is a little too clever for my taste. I'd probably prefer `if any([cleaned_data.risk_internal_only, cleaned_data.risk_partner_related, cleaned_data.risk_brand, ...])`.